app/scripts/models/application-settings.coffee

```diff
@@ -22,6 +23,7 @@ define [
   class ApplicationSettingsModel extends BaseRelationalModel
 
     initialize: ->
+      @set('fielddbApplication' , FieldDB.FieldDBObject.application)
       @listenTo Backbone, 'authenticate:login', @authenticate
```

Comment:
@jrwdunham what is the syntax to do this line so we can access the fielddbapplication later via
globals.applicationSettings.fielddbApplication 
or 
globals.applicationSettings.get('fielddbApplication')